tsl/test/sql/partialize_finalize.sql

```diff
@@ -296,3 +296,44 @@ select _timescaledb_internal.finalize_agg( 'aggregate_to_test_ffunc_extra(int, a
 
 with cte as (SELECT  _timescaledb_internal.partialize_agg(aggregate_to_test_ffunc_extra(8, 1::bigint)) as part)
 select _timescaledb_internal.finalize_agg( 'aggregate_to_test_ffunc_extra(int, anyelement)', null, null, array[array['pg_catalog'::name, 'int4'::name], array['pg_catalog', 'int8']], part, null::text) is null from cte;
+
+
+-- Issue 4922
+CREATE TABLE issue4922 (
+  time  TIMESTAMPTZ NOT NULL,
+  value INTEGER
+);
+
+SELECT create_hypertable('issue4922', 'time');
+
+INSERT INTO issue4922 (time, value)
+SELECT time, (random()*30) as value
+FROM generate_series(now() - interval '100 hours', now(), '1 sec') AS time;
+
+SET force_parallel_mode = 'on';
+SET parallel_setup_cost = 0;
+
+-- Compare the results (should return NO ROWS)
+(SELECT
```

Comment:
Slightly confused by this test -- which subselect is the broken one? Maybe we could generate stable pseudo-random data and just show the results of both subselects. For pseudo-random we can use something like this: https://github.com/timescale/timescaledb/blob/main/tsl/test/sql/dist_param.sql#L20-L33

Reply:
Also might be good to have an explain query so that we are absolutely sure it runs in parallel, because parallel execution is notoriously hard to achieve sometimes.

Reply:
You're correct will update the tests.

Reply:
Tests updated